src/server.ts

```diff
@@ -90,31 +99,56 @@ function throwResponseValidationError(tag: string, originalValue: any, e: oar.Ma
   return null as any;
 }
 
-function voidify(value: {} | undefined) {
-  if (value == null || Object.keys(value).length > 0) {
+function lowercaseObject(o?: {} | undefined | null) {
```

Comment:
Don't need `undefined` here as `o` is optional. Maybe remove the `null` as well. Why would `headers` ever be `null` here?